### DISCUSSION TITLE
What does "you must enable use of your device camera to capture your image." mean?

### DISCUSSION THREAD

**Discussion by ThDag** [community/community#58990] [GitHub Education]
### Hi everyone,

I am trying to submit for GitHub education but my school doesn't provide School ID or School Email so i uploaded my student documents thats used across the country as legal proof of education.

My request was refused with the following response:

> Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Jun 23, 2023 for [@ThDag](https://github.com/ThDag).
>
> **If your school does not provide academic email then you must enable use of your device camera to capture your image.**

Considering there is no way to proof the documents belong to me,
Do i have to take a picture of myself with the documents to proof its me in the document ?

Any suggestion is welcome.

**Comment by GutowskiO** [comment]
I have the same problem bro

**Reply by ThDag** [reply]
hey man i posted a selfy with the paper and it got certified o request that

**Comment by samus-aran** [comment]
Hi @ThDag 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!

**Reply by ThDag** [reply]
Thanks for the response i have solved the issue with taking picture of the certificate with my face and i am happily a member of the student program you can close the thread.
